cluster/kube-down.sh

```diff
@@ -28,6 +28,18 @@ echo "Bringing down cluster using provider: $KUBERNETES_PROVIDER"
 
 verify-prereqs
 teardown-monitoring
+
+# If logging with Fluentd to Elasticsearch was enabled then
+# delete the pods and service and remove the firewall rule
+# for accessing Elasticsearch and Kibana.
+if [[ "${ENABLE_NODE_LOGGING-}" == "true" ]]; then
```

Comment:
I thought it made sense to have the predicate outside since the function that "tears down logging" because I thought this should only be called if logging was enabled in the first place. OK: I shall push the predicate into the function (which makes it similar in style to how the monitoring code is written). But I am sure some software engineering pedant would have something to say about this...